lib/helper.js

```diff
@@ -170,10 +175,68 @@ async function deleteAllLabels(token, owner, host, repository, cli) {
 
     // Done
     if (cli.flags.uploadLabels) echo.success('Done!\n')
-    else {
+    else if (exit) {
         console.log()
-        echo.success('Finished!', true)
+        echo.success('Finished!', exit)
+    }
+}
+
+// Gets array of repository names in "owner" organization
+async function getRepositories(token, owner, host, cli) {
+    // Variables
+    let repos = []
+    let pageCount = 0
+    const perPage = 100
+    
+    // Check required Flags (repository not needed here but need to send so check won't fail)
+    checkRequiredFlags(token, owner, "sample")
+
+    // Get headers for a request to list all repositories for the given org
+    let response = await axios.headRepoList(true, token, owner, host, 1)
+    if (response.status != 200 || !response.headers.link) echo.error('Unexpected status or response on HEAD request for GHE repository list.', true)
+
+    // By passing in 1 as the per_page value, the "last" page link will have a page number equal to the number of repos
+    // To compute the actual number of necessary requests, divide the number of repos by the requested per_page limit (max 100)
+    let numberOfRepos = getPageCountFromLinkHeader(response.headers.link)
+    pageCount = Math.ceil(numberOfRepos / perPage)
+
+    echo.info(chalk.bold('Fetching list of ' + numberOfRepos + ' repository name(s) across ' + pageCount + ' page(s), in organization ' + owner + '...'))
+
+    for (let i=1; i<=pageCount; i++) {
+        let res = await axios.getReopsByPage(true, token, owner, host, i, perPage)
+        res.data.forEach(repo => {
+            // Get the url for each repo object and extract just the name at the end of the url
+            repos.push(repo.html_url.substring(repo.html_url.lastIndexOf('/') + 1))
+        });
+    }
+
+    return repos
+}
+
+// Gets the total number of pages in the list of repositories by parsing the "last" rel of the "link" response header
+// GitHubApi has a per_page limit of 100, default 30
+function getPageCountFromLinkHeader(header) {
+    if (!header || header.length === 0) {
+        echo.error('header input must not be null or of zero length', true)
+    }
+
+    // Split parts by comma
+    var parts = header.split(',')
+
+    // Parse each part into a named link
+    for(var i=0; i<parts.length; i++) {
+        var section = parts[i].split(';')
+        if (section.length !== 2) {
+            echo.error('section could not be split on ";"', true)
+        }
```

Comment:
Personal preference: `if` one-liner
```suggestion
        if (section.length !== 2) echo.error('section could not be split on ";"', true)
```